.readthedocs.yml

```diff
@@ -4,10 +4,13 @@
 
 version: 2
 
+conda:
+  environment: documentation/conda.yml
```

Comment:
Please add a comment indicating _why_ we use conda in this file, mentioning `--enable-shared`